test/Test/Universum/Property.hs

```diff
@@ -0,0 +1,107 @@
+
+module Test.Universum.Property
+        ( hedgehogTestTree
+        ) where
+
+import Universum
+import Hedgehog
+import Test.Tasty
```

Comment:
We should have imports either `qualified` or with explicit imports lists.

Reply:
done. I added Universum as a qualified import also, only because this is Universum testing code, and I think in some places it's good to explicitly label that e.g. `and` is `U.and`